src/util/gcode.py

```diff
@@ -0,0 +1,43 @@
+from pygcode import (GCode, GCodeRapidMove, GCodeLinearMove, GCodeFeedRate, GCodeAbsoluteDistanceMode,
+                     GCodeUseMillimeters, Word)
+from constants import START_POSITION, FEED_RATE, Z_AXIS_CUT_POSITION
+
+# Scribe lines are in the format:
+# [[[Xstart0, Ystart0], [Xdest0, Ydest0]], [[Xstart1, Ystart1], [Xdest1, Ydest1]]...]
+
+
+def parse_gcode(scribe_lines):
+    """
+    Parse the cut coordinates and return a list of GCode objects.
+    """
+    g_codes = []
+
+    for line in scribe_lines:
+        start_coordinates = list(map(_meters_to_millimeters, line[0]))
+        dest_coordinates = list(map(_meters_to_millimeters, line[1]))
```

Comment:
No caso dessa maneira não atende aos casos aonde tem um seguimento com mais de 2 pontos, por exemplo, um corte em V. Ou devemos fazer uma reta, sobe agulha, desce agulha, outra reta? Mesmo que o início da próxima reta seja o final da anterior (ou seja, a agulha não precisa se movimentar)